ui/settings_page.go

```diff
@@ -123,17 +140,21 @@ func (pg *settingsPage) general() layout.Widget {
 				}),
 				layout.Rigid(pg.lineSeparator()),
 				layout.Rigid(func(gtx C) D {
-					return pg.subSection(gtx, "Currency conversion", func(gtx C) D {
-						return layout.Flex{}.Layout(gtx,
-							layout.Rigid(func(gtx C) D {
-								txt := pg.theme.Body1("None")
-								txt.Color = pg.theme.Color.Gray
-								return txt.Layout(gtx)
-							}),
-							layout.Rigid(func(gtx C) D {
-								return pg.currencyConversion.Layout(gtx)
-							}),
-						)
+					return decredmaterial.Clickable(gtx, pg.currencyConversion.clickable, func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPaddingMinus5}.Layout(gtx, func(gtx C) D {
+							return pg.subSection(gtx, "Currency conversion", func(gtx C) D {
+								return layout.Flex{}.Layout(gtx,
+									layout.Rigid(func(gtx C) D {
+										txt := pg.theme.Body2("None")
+										txt.Color = pg.theme.Color.Gray
+										return txt.Layout(gtx)
+									}),
+									layout.Rigid(func(gtx C) D {
+										return pg.currencyConversion.icon.Layout(gtx)
+									}),
+								)
+							})
+						})
```

Comment:
This code block seems to be reused for a number of option rows, you can consider making it a function or method and reusing that.